docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,88 @@
+
+!!! INFO "CSM -- Quick Definition"
+    The CSM is a standard that defines an API for sensor models, thereby providing a common structure that guarantees interoperability of any sensor models that adhere to the standard.
+
+## The ASC Sensor Model Ecosystem
+The Astrogeology Science Center has created and maintains a suite of software packages that provide end-to-end processing for sensor models, including a metadata specification, CSM-compliant sensor model implementation, and tools to leverage those sensor models for scientific processing.
+
+
+### Abstraction Library for Ephemerides (ALE)
+The [Abstraction Library for Ephemerides](https://github.com/DOI-USGS/ale/) (ALE) provides the tools and information necessary to derive and access a sensor's exterior orientation.  ALE provides a suite of instrument-specific drivers that combine information from a variety of metadata formats (image labels), sensor types, and data sources into a single CSM compliant format. It is important to note that ALE is responsible for abstracting away intermediate reference frames between the body-centered, body-fixed frame and the sensor frame so that the resulting model can transform directly from the sensor frame to the BCBF frame.
+
+ALE uses a collection of drivers and class mixins to provide ISDs for a variety of sensor models.  [Drivers](https://github.com/DOI-USGS/ale/tree/main/ale/drivers) for many major missions are available in ALE.  Each driver is required to provide support for at least one label type (ISIS or PDS3) and one source of SPICE data (ISIS or NAIF). It is important to note that not all drivers currently support all combinations of labels and SPICE data sources.
+
+!!! INFO "ALE -- Quick Definition"
+    ALE is a software package that relies on a collection of instrument-specific drivers in order to calculate and provide access to camera exterior orientations.  This information is output in a standard ISD format that provides all the information required to instantiate a CSM sensor model.
+
+### Integrated Software for Imagery and Spectrometry (ISIS)
+[ISIS](https://github.com/DOI-USGS/ISIS3) is a software suite that comprises a variety of image analysis tools ranging from traditional tools like contrast, stretch, image algebra, and statistical analysis tools to more complex processes such as mosaicking, photometric modeling, noise reduction, and bundle adjustment. In addition to these analysis tools, ISIS defines the .cub format which can capture 3-dimensional image data (lines, samples, and bands) as well as image metadata in the form of an image label.  In addition to capturing image data and metadata, .cub files can be augmented with camera geometry information such that elements of the camera model are captured alongside the data.
+
+#### Bundle Adjustment / Jigsaw
```

Comment:
I get why this is here, but this is 100% not a sensor model 'thing'. Sure it uses sensor models all over the place, but so does stuff like map projection. What is the rationale for including this topic, in this short form, here?

Reply:
The idea was to use bundle adjustment as a demonstration of how sensor model geometry can change over time.  The portion on statefulness of sensor models mentions bundle adjustment, and I needed something to point to since we don't have existing bundle adjust docs.

More of a placeholder until we have something better to point to.